src/wp-includes/theme.php

```diff
@@ -4416,3 +4445,45 @@ static function ( $active, WP_Customize_Panel $panel ) {
 		2
 	);
 }
+
+/**
+ * Loads view transitions for the current theme, if configured.
+ *
+ * @since 6.9.0
+ */
+function wp_load_view_transitions() {
+	if ( ! current_theme_supports( 'view-transitions' ) ) {
+		return;
+	}
+
+	$stylesheet = file_get_contents( ABSPATH . WPINC . '/view-transitions.css' );
+
+	// Use an inline style to avoid an extra request, also because it's just 1 line of CSS.
+	wp_register_style( 'wp-view-transitions', false, array(), null );
+	wp_add_inline_style( 'wp-view-transitions', $stylesheet );
+	wp_enqueue_style( 'wp-view-transitions' );
+
+	$theme_support = get_theme_support( 'view-transitions' );
+
+	// No point in loading the script if no specific view transition names are configured.
+	if ( ! $theme_support['global-transition-names'] && ! $theme_support['post-transition-names'] ) {
+		return;
+	}
+
+	$config = array(
+		'postSelector'            => $theme_support['post-selector'],
+		'globalTransitionNames'   => $theme_support['global-transition-names'],
+		'postTransitionNames'     => $theme_support['post-transition-names'],
+		'chronologicalSlideInOut' => $theme_support['chronological-slide-in-out'],
+	);
+	$script = file_get_contents( ABSPATH . WPINC . '/view-transitions.js' );
```

Comment:
```suggestion
	$script = file_get_contents( ABSPATH . WPINC . '/view-transitions' . wp_scripts_get_suffix() . '.js' );
```

Although this won't work until the script is moved into `js/_enqueues`.

Reply:
See https://github.com/WordPress/wordpress-develop/pull/8370#discussion_r1980176554

Reply:
Updated in https://github.com/WordPress/wordpress-develop/pull/8370/commits/816315910a9147899a5cff75cdcdb092dadffeb8